jest.config.js

```diff
@@ -1,8 +1,15 @@
+const path = require("path");
+
 module.exports = {
   testEnvironment: "jest-environment-jsdom",
   testRegex: "src/.+\\.(test|spec)\\.[jt]sx?$",
-  setupFilesAfterEnv: ["./jest.setup.spec.js"],
+  setupFilesAfterEnv: ["./jest/setup.ts"],
```

Comment:
Setup file should be a ts file when running typescript. Otherwise a whole bunch of warnings is logged